blaze-core/src/main/scala/org/http4s/internal/SSLContextOption.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2014 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s.internal
```

Comment:
Had to bring this because of this refactoring https://github.com/http4s/http4s/pull/7175/files in core...

Reply:
I'm not sure this repo should be writing to `org.http4s.internal`.  How about `org.http4s.blaze.internal`?

Reply:
Good call! Will fix this